api/integrations/slack/slack.py

```diff
@@ -0,0 +1,62 @@
+import typing
+
+from django.conf import settings
+from slack_sdk import WebClient
+from slack_sdk.http_retry.builtin_handlers import RateLimitErrorRetryHandler
+
+from integrations.common.wrapper import AbstractBaseEventIntegrationWrapper
+
+
+class SlackWrapper(AbstractBaseEventIntegrationWrapper):
+    def __init__(self, api_token: str, channel_id: str):
+        self.client = get_client(api_token)
+        self.channel_id = channel_id
+
+    def _track_event(self, event: dict) -> None:
+        self.client.chat_postMessage(channel=self.channel_id, text=event["text"])
+
+    @staticmethod
+    def generate_event_data(log: str, email: str, environment_name: str) -> dict:
+        return {
+            "text": f"{log} by user {email}",
+            "title": "Flagsmith Feature Flag Event",
+            "tags": [f"env:{environment_name}"],
+        }
+
+
+def get_bot_token(code: str, redirect_uri: str) -> str:
```

Comment:
I think it's because the code that invokes these methods don't have access to all the variables required to init the class
in this case ` api_token: str, channel_id: str` , we can make it static method tho?